libs/blocks/text/text.css

```diff
@@ -73,6 +73,25 @@
   margin: var(--spacing-m) 0;
 }
 
+.text-block .title-l {
+  font-size: var(--type-body-m-size);
+  line-height: var(--type-body-m-lh);
+  font-weight: bold;
+  text-transform: none;
+  margin-bottom: var(--spacing-xs);
+}
+
+.text-block :is(.image, .lockup-area) {
+  margin-block-end: var(--spacing-m);
+}
+
+.text-block .image em {
```

Comment:
Is this supporting a specific authoring pattern? It seems kind of unlikely to expect an image in an <em> tag?

Reply:
@ryanmparrish Assuming you're referring to the `em` in this rule -- yes, this is actually how lockup text is being handled in other blocks currently. However, `.image` overrides that and expects an `<em>` tag to achieve an image caption as detailed in the [Figma designs](https://www.figma.com/design/eUM6feFsRokh4uxM3vZFL3/Basic-Blocks?node-id=450-12849&node-type=FRAME&t=w9DHQD809VEgLP67-0)

Reply:
FWIW, the authoring pattern behind this (and those other lockup implementations) is just adding text after an image _without_ a line break. In this block, it'll only apply lockup styles if you add a `-lockup` variant to the block.

Reply:
Hmm, This sounds like a use for the [[figure](https://milo.adobe.com/docs/library/blocks/figure)] block on top of a text block rather? Was this a requirement for the text block or a layout? I'd like to be careful bloating the text block w/ too many features if possible.

Reply:
Very good point! I'll bring that back to the stakeholders and see what they say 👍

Reply:
Worked this out over Slack with everyone involved, but leaving this here to document it:

Apparently we are looking at deprecating the Figure block, because it does not respect Milo defaults (alignment, dark mode support, etc). That is probably a discussion for another time, but for now (since it lacks the aforementioned features), the general direction for this work is that we support this in the Text Block so that it is a more globally supported option.